corehq/apps/cloudcare/views.py

```diff
@@ -105,27 +107,15 @@ class FormplayerMain(View):
     def dispatch(self, request, *args, **kwargs):
         return super(FormplayerMain, self).dispatch(request, *args, **kwargs)
 
-    def fetch_app(self, domain, app_id):
-        return _fetch_build(domain, self.request.couch_user.username, app_id)
+    def fetch_app_fn(self):
```

Comment:
Maybe make this a property?
```suggestion
    @property
    def fetch_app_fn(self):
```

This will obviously require a few other changes as well if accepted.

Reply:
So I initially did this, and ran into an issue in tests related to attempting to pickle something when this was a property, which I didn't feel like digging into too much since sticking with a method sounded easier.